@xen-orchestra/web/src/actions/vm-actions.ts

```diff
@@ -0,0 +1,20 @@
+import type { XoHost, XoVm } from '@vates/types'
+import { useFetch } from '@vueuse/core'
+
+async function vmAction(vmId: XoVm['id'], action: string, body?: Record<string, any>): Promise<void> {
+  const { error } = await useFetch(`/rest/v0/vms/${vmId}/actions/${action}`, {
```

Comment:
Async actions are not supported for now in XO6.
I will do a PR soon for that
```suggestion
  const { error } = await useFetch(`/rest/v0/vms/${vmId}/actions/${action}?sync=true`, {
```